graphitepager/worker.py

```diff
@@ -120,11 +120,11 @@ def run(args):
                     http_connect_timeout_s_ = http_connect_timeout_s,
                     http_read_timeout_s_    = http_read_timeout_s,
                 )
-            except requests.exceptions.RequestException as e:
+            except (ValueError, requests.exceptions.RequestException) as e:
                 if not alert.alert_data['allow_no_data']:
                     print "Error, {0}".format(alert.alert_data)
                     update_notifiers_missing(notifier_proxy, alert, config)
-                print "Exception: %s" % e
+                print "Exception in %s: %s" % (alert.get('name'),e)
```

Comment:
We'll catch more info in each such report, help alert authors narrow in on the problem.